src/molara/structure/io/importer.py

```diff
@@ -351,7 +349,11 @@ def __init__(self, path: PathLike | str) -> None:
 
         :param path: input file path
         """
-        import cclib
+        try:
+            import cclib
+        except ImportError as err:
+            msg = "Could not import cclib."
+            raise FileFormatError(msg) from err
```

Comment:
Why `FileFormatError`? Shouldn't this raise `ModuleNotFoundError` or `PackageNotFoundError`? (I'm not quite sure what the difference between those two is)?

Reply:
So after a quick search it seems like `ModuleNotFoundError` is a subclass of `ImportError` and `PackageNotFoundError` is then a subclass of `ModuleNotFoundError`.

Reply:
I think this is for some reason how we have also handled it with pymatgen, so I wanted to keep it consistent. I agree, however, that `FileFormatError` is not ideal

Reply:
I have now changed FileFormatError to ImportError @ab5424.